tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/mappingconstructor/MappingConstructorExprTest.java

```diff
@@ -419,6 +419,30 @@ public void testDuplicateFieldWithEscapeSequence() {
         Assert.assertEquals(compileResult.getErrorCount(), index);
     }
 
+    @Test
```

Comment:
We usually add tests with custom module imports as bala tests.

This is also reproducible without an import, right? With an undefined symbol for example. If so, we can stick to a single file test.